apiary-metastore-events/kafka-metastore-events/pom.xml

```diff
@@ -0,0 +1,104 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.expediagroup.apiary</groupId>
+    <artifactId>apiary-metastore-events-parent</artifactId>
+    <version>4.2.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>kafka-metastore-events-parent</artifactId>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>kafka-metastore-listener</module>
+    <module>kafka-events-common</module>
+  </modules>
+
+  <properties>
+    <commons.lang3.version>3.7</commons.lang3.version>
+    <hadoop.version>2.7.2</hadoop.version>
+    <hcommon.hive.metastore.version>1.0.0</hcommon.hive.metastore.version>
+    <hive.version>2.3.0</hive.version>
```

Comment:
I think this will be set way up in the top level parent so you don't need to set it here.